### DISCUSSION TITLE
backlight setting

### DISCUSSION THREAD

**Discussion by InsideMyEye** [BartoszCichecki/LenovoLegionToolkit#1192] [Q&A]
does anyone know how to fix this keyboard backlight issue, i set different color for three preset of my backlight setting, but when i shutdown my laptop at preset 3 and then i start my laptop, the backlight setting automatically back to preset 1, or if turn off my backlight before shutdown my laptop and then when i turn on my laptop, the backlight keyboard automatically turn on at  preset 1 setting.

**Comment by BartoszCichecki** [comment]
Your chances of getting any help without even mentioning which laptop you have are pretty slim.

**Reply by InsideMyEye** [reply]
my laptop model is lenovo lOQ 15IRX9
